src/BigQueryConnection.php

```diff
@@ -43,11 +52,14 @@ public function executeQuery(string $query): QueryResults
     {
         $queryOptions = $this->session->getAsQueryOptions();
         $queryOptions['configuration']['labels'] = ['run_id' => $this->runId];
+        if ($this->queryTimeout !== 0) {
+            $queryOptions['configuration']['jobTimeoutMs'] = $this->queryTimeout * 1000;
+        }
```

Comment:
správný způsob jak nastavit timeout na query https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#jobconfiguration akorát nastavit to správně pro klienta je poser